src/pages/workspace/reimburse/WorkspaceRateAndUnitPage.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import {Keyboard, View} from 'react-native';
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import ONYXKEYS from '../../../ONYXKEYS';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import styles from '../../../styles/styles';
+import compose from '../../../libs/compose';
+import * as Policy from '../../../libs/actions/Policy';
+import CONST from '../../../CONST';
+import Picker from '../../../components/Picker';
+import TextInput from '../../../components/TextInput';
+import WorkspacePageWithSections from '../WorkspacePageWithSections';
+import withPolicy, {policyPropTypes, policyDefaultProps} from '../withPolicy';
+import {withNetwork} from '../../../components/OnyxProvider';
+import OfflineWithFeedback from '../../../components/OfflineWithFeedback';
+import Form from '../../../components/Form';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+
+const propTypes = {
+    ...policyPropTypes,
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    ...policyDefaultProps,
+};
+
+class WorkspaceRateAndUnitPage extends React.Component {
+    constructor(props) {
+        super(props);
+        const distanceCustomUnit = _.find(lodashGet(props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        const customUnitRate = _.find(lodashGet(distanceCustomUnit, 'rates', {}), rate => rate.name === 'Default Rate');
+
+        this.state = {
+            unitID: lodashGet(distanceCustomUnit, 'customUnitID', ''),
```

Comment:
Why are we using `state` here? There are no setState calls, we aren't updating these values

Reply:
I removed the state and moved the initialization when we need the variables